action.yaml

```diff
@@ -30,10 +35,12 @@ inputs:
     required: false
     type: integer
 outputs:
-  created_pages:
+  urls_with_actions:
     description: |
-      A JSON map with the pages that have been created where the key is the
-      local path and the value is the URL to the created page.
+      A JSON map with the urls and the actions that have been taken against
+      them.
+  index_url:
+    description: The URL to the documentation index page.
```

Comment:
Is this the base URL? Can we provide a clearer description?

Reply:
It is the URL that should be set under the `docs` key in `metadata.yaml`. Maybe `The URL to the documentation index page on discourse. The docs key in metadata.yaml should be updated to this value after the first run.`?